code/game/objects/structures/crates_lockers/crates/wooden.dm

```diff
@@ -15,4 +15,116 @@
 	new	/obj/item/reagent_containers/food/drinks/soda_cans/canned_laughter(src)
 	new /obj/item/pneumatic_cannon/pie(src)
 	new /obj/item/reagent_containers/food/snacks/pie/cream(src)
-	new /obj/item/storage/crayons(src)
\ No newline at end of file
+	new /obj/item/storage/crayons(src)
+
+/obj/structure/closet/crate/wooden/artifact/PopulateContents()
+	. = ..()
+	switch (pickweight(list("worldwar" = 3, "tommy" = 2, "gang" = 1, "rushn" = 2, "detective" = 1, "wiz_charge" = 2, "wiz_summon_forcewall" = 2, "wiz_blind" = 1, "wiz_sacredflame" = 1, "wiz_barnyard" = 1, "wiz_knock_smoke" = 1, "wiz_door" = 2, "wiz_fireball" = 1, "wiz_suit" = 2, "cutlass" = 2, "katana" = 1, "pharaoh" = 2, "knight" = 1, "bo" = 2, "hook" = 1, "spiders" = 2, "tiki" = 1)))
+		if("worldwar")
+			new /obj/item/clothing/suit/armor/vest/old(src)
+			new /obj/item/gun/ballistic/rifle/boltaction(src)
+			new /obj/item/grenade/syndieminibomb/concussion(src)
+			new /obj/item/clothing/head/helmet/old(src)
+
+		if("tommy")
+			new /obj/item/gun/ballistic/automatic/tommygun/no_mag(src)
+			new /obj/item/ammo_box/magazine/tommygunm45/empty(src)
+			for(var/i = 0, i < 4, ++i)
+				new /obj/item/ammo_casing/c45(src)
+			new /obj/item/clothing/head/fedora(src)
+
+		if("gang")
+			new /obj/item/gun/ballistic/automatic/mini_uzi(src)
+
+		if("rushn")
+			new /obj/item/gun/ballistic/automatic/pistol/APS(src)
+			new /obj/item/switchblade(src)
+			new /obj/item/clothing/head/ushanka(src)
+
+		if("detective")
+			new /obj/item/gun/ballistic/revolver/detective(src)
```

Comment:
this is also pretty underwhelming for 12 TC

Reply:
It's not meant to be guaranteed worth the price. Coming with a free baton, it's probably still underpriced though since the revolver is close to as good as the stechkin. Maybe throwing in some syndicate smokes might even it out a bit.

Reply:
The detective revolver is second only to the actual revolver in terms of round-start traitor mode weapons.

Reply:
detective revolver can be screwdrivered to take .357 rounds, so it's pretty much the same thing.